package.json

```diff
@@ -72,11 +72,12 @@
     },
     "keywords": [
         "hazelcast",
-        "nodejs",
+        "real-time",
```

Comment:
minor: is it `real-time` or `real time`, as we have used the latter above, and the former here

Reply:
I used real-time because it was real-time in hazelcast.com. I will make both real-time